public/app/features/plugins/built_in_plugins.ts

```diff
@@ -110,7 +109,6 @@ const builtInPlugins: any = {
   'app/plugins/panel/canvas/module': canvasPanel,
   'app/plugins/panel/icon/module': iconPanel,
   'app/plugins/panel/dashlist/module': dashListPanel,
-  'app/plugins/panel/pluginlist/module': pluginsListPanel,
```

Comment:
if you do this you have to remove all the plugin files, as it wont load if this is removed